slides/intro-04-evaluating-models.qmd

```diff
@@ -259,6 +259,70 @@ augment(forested_fit, new_data = forested_train) %>%
 countdown::countdown(minutes = 5, id = "roc-curve")
 ```
 
+## Brier score
+
+What if we don't turn predicted probabilties into class predictions?
+
+. . .
+
+The Brier score is analogous to the mean squared error in regression models:
+
+$$
+Brier_{class} = \frac{1}{N}\sum_{i=1}^N\sum_{k=1}^C (y_{ik} - \hat{p}_{ik})^2
+$$
+
+## Brier score
+
+```{r brier}
+augment(forested_fit, new_data = forested_train) %>% 
+  brier_class(truth = forested, .pred_Yes) 
+```
+
+. . .
+
+Smaller values are better, for binary classification the "bad model threshold" is about 0.25.
```

Comment:
Ref for `0.25`: https://www.tidymodels.org/learn/models/calibration/#a-naive-bayes-model

Reply:
Just read that linked section--what's the explanation for why .25? Should we drop it in a `:::notes`?

Reply:
That's a good question -- @topepo do you have some context for us here?

Reply:
I can add an annotation. For two classes (under certain conditions) the worst possible model predicts everything to be 1/2, so it is (1 - 1/2)^2

Reply:
Add to this PR or add another?

Reply:
Added it here

Reply:
Nice one, thank you! 🙌